servers/src/main/java/tachyon/master/file/meta/Inode.java

```diff
@@ -278,6 +295,51 @@ public synchronized void setPinned(boolean pinned) {
     mPinned = pinned;
   }
 
+  /**
+   * @return the username of the inode
+   */
+  public synchronized String getUsername() {
+    return mUsername;
+  }
+
+  /**
+   * Sets the username of the inode
```

Comment:
end sentence with a period

insert a blank line between the method-level comment and parameter descriptions